src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -306,6 +308,52 @@ private void RecordingGUI()
             }
         }
 
+        private void OcclusionSettingsGUI()
+        {
+            occlusionSettingsFoldout = EditorGUILayout.Foldout(occlusionSettingsFoldout, "Occlusion Settings");
+            if (occlusionSettingsFoldout)
+            {
+                CompositionManager compositionManager = GetCompositionManager();
+                bool running = compositionManager != null && compositionManager.TextureManager != null && compositionManager.TextureManager.videoFeedColorCorrection != null;
+                if (running)
+                {
+                    if (!compositionManager.IsOcclusionSettingSupported(compositionManager.OcclusionMode))
+                    {
+                        RenderTitle("The current camera does not support occlusion.", Color.clear);
+                    }
+                    else if (compositionManager.OcclusionMode == OcclusionSetting.BodyTracking)
```

Comment:
Any reason why to not include min/max restrictions when body tracking? I think this could be a useful feature if the camera is pointed outward to a crowd and you only want it to pick up people within a certain distance. Should work with same implementation you have below for RawDepthCamera occlusion mode.

Reply:
I will remove this min and max logic. Blur was the only parameter that made a substantial difference.